src/main/scala/firrtl/options/OptionParser.scala

```diff
@@ -28,7 +28,7 @@ trait ExceptOnError { this: OptionParser[_] =>
   */
 trait DuplicateHandling extends OptionParser[AnnotationSeq] {
 
-  override def parse(args: Seq[String], init: AnnotationSeq): Option[AnnotationSeq] = {
+  override def parse(args: scala.collection.Seq[String], init: AnnotationSeq): Option[AnnotationSeq] = {
```

Comment:
maybe unrelated too?

Reply:
No. `Seq` used to be `scala.collection.Seq`, now it is `scala.collection.immutable.Seq`. Since we extend the `OptionParser` we need to be explicit about overriding the correct `Seq`.

Reply:
wooo, type system of 2.13 is amazing!